### PR TITLE
prevent bash expansion in password var

### PR DESCRIPTION
**Pull request by clnperez**
Single quotes will keep any bash expansion from being done inside a string. 
If you have any special bash characters in your password, the rh
subscription fails in your bastion node and no services will be
available for your cluster.

Signed-off-by: Christy Norman <christy@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#" title="Author self-approved">clnperez</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpradipt**
You can assign the PR to them by writing `/assign @bpradipt` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by clnperez** [comment]
I submitted this against master instead of the 4.5 branch even though it's the default branch -- assuming it'll get cherry-picked in. Or I can submit a 2nd PR to that branch if this one's good to go.

**Comment by yussufsh** [comment]
/lgtm
This can be cherry picked. Thanks @clnperez .

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#" title="Author self-approved">clnperez</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#issuecomment-694631665" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yussufsh — APPROVED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#" title="Author self-approved">clnperez</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#pullrequestreview-491150273" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @clnperez

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#pullrequestreview-491156778" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#" title="Author self-approved">clnperez</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#pullrequestreview-491150273" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by mkumatag — APPROVED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#pullrequestreview-491156778" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#" title="Author self-approved">clnperez</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#pullrequestreview-491158412" title="Approved">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#pullrequestreview-491150273" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@yussufsh: The label(s) `kind/cherry-pick-approved` cannot be applied, because the repository doesn't have them

<details>

In response to [this](https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/78#issuecomment-694814828):

>/kind cherry-pick-approved


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
